app/parse.py

```diff
@@ -8,9 +14,131 @@ class Quote:
     tags: list[str]
 
 
-def main(output_csv_path: str) -> None:
-    pass
+@dataclass
+class Author:
+    name: str
+    birth_date: str
+    birth_location: str
+    decription: str
```

Comment:
There is a typo in the `Author` dataclass. The field `decription` should be `description` to match the intended use in the `parse_single_author` function.